cli/integrity/integrity_test.go

```diff
@@ -0,0 +1,62 @@
+package integrity_test
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/integrity"
+)
+
+func TestNewSigner(t *testing.T) {
+	testCases := []struct {
+		name           string
+		privateKeyPath string
+	}{
+		{
+			name:           "Empty path",
+			privateKeyPath: "",
+		},
+		{
+			name:           "Arbitrary path",
+			privateKeyPath: "private.pem",
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			signer, err := integrity.NewSigner(testCase.privateKeyPath)
+			require.Nil(t, signer, "signer must not be created")
+			require.Error(t, err, "an error should be produced")
+		})
+	}
+}
+
+func TestRegisterWithIntegritySigner(t *testing.T) {
+	someStr := ""
+
+	testCases := []struct {
+		name    string
+		command *cobra.Command
+		dst     *string
+	}{
+		{
+			name:    "Empty dst",
+			command: &cobra.Command{},
+			dst:     nil,
+		},
+		{
+			name:    "Nothing empty",
+			command: &cobra.Command{},
+			dst:     &someStr,
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			integrity.RegisterWithIntegrityFlag(testCase.command.Flags(), testCase.dst)
```

Comment:
It seems to me that there is an extra command object here.

Reply:
Fixed

Reply:
I suppose a FlagSet could be created without command + we could test for a `nil` value.

Reply:
Oh, I see what you've meant now. I've updated the tests.